src/SprykerMiddleware/Zed/Process/Business/Iterator/IteratorFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Business\Iterator;
+
+use Iterator;
+use Spryker\Zed\Kernel\Business\AbstractBusinessFactory;
+
+class IteratorFactory extends AbstractBusinessFactory
```

Comment:
It's not a part of the Spryker infrastructure, but a standalone factory. So it should not extend Spryker business factory.